src/renderer/components/+extensions/extensions.tsx

```diff
@@ -158,13 +164,14 @@ export class Extensions extends React.Component {
       if (InputValidators.isUrl.validate(installPath)) {
         const { promise: filePromise } = downloadFile({ url: installPath, timeout: 60000 /*1m*/ });
         const data = await filePromise;
-        this.requestInstall({ fileName, data });
+        await this.requestInstall({ fileName, data });
       }
       // otherwise installing from system path
       else if (InputValidators.isPath.validate(installPath)) {
-        this.requestInstall({ fileName, filePath: installPath });
+        await this.requestInstall({ fileName, filePath: installPath });
       }
     } catch (error) {
+      this.startingInstall = false;
```

Comment:
Probably should be within `finally`-block.

Reply:
finally wouldn't work, because we could reach that before the pre-install phase is completed.